File: app/src/main/java/vn/loitp/app/activity/customviews/answerview/AnswerViewActivity.java
package vn.loitp.app.activity.customviews.answerview;

import android.os.Bundle;
import android.widget.LinearLayout;

import com.core.base.BaseFontActivity;
import com.views.LToast;
import com.views.answerview.AnswerView;

import loitp.basemaster.R;

public class AnswerViewActivity extends BaseFontActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //use xml
        useXML();

        //use JAVA codes
        useJava();
    }

    private void useXML() {
        AnswerView answerView1 = (AnswerView) findViewById(R.id.av_1);
        answerView1.setNumber(1);
        answerView1.setOnAnswerChange(new AnswerView.OnAnswerChange() {
            @Override
            public void onAnswerChange(AnswerView view, int index) {
                LToast.INSTANCE.show(activity, "Click: " + index);
            }
        });
        answerView1.setActiveChar('A');
        //answerView1.resize(2);

        AnswerView answerView2 = (AnswerView) findViewById(R.id.av_2);
        answerView2.setNumber(2);
        answerView2.setOnAnswerChange(new AnswerView.OnAnswerChange() {
            @Override
            public void onAnswerChange(AnswerView view, int index) {
                LToast.INSTANCE.show(activity, "Click: " + index);
            }
        });
    }

    private void useJava() {
        LinearLayout ll = (LinearLayout) findViewById(R.id.ll);
        for (int i = 0; i < 10; i++) {
            AnswerView answerView = new AnswerView(activity);
            answerView.init(i + 3, 6, true, true, true, true);
            answerView.setOnAnswerChange(new AnswerView.OnAnswerChange() {
                @Override
                public void onAnswerChange(AnswerView view, int index) {
                    LToast.INSTANCE.show(activity, "Click: " + index);
                }
            });
            ll.addView(answerView);
        }
    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_answer_view;
    }
}

File: app/src/main/java/vn/loitp/a/cv/answerView/AnswerViewActivity.kt
package vn.loitp.a.cv.answerView

import android.os.Bundle
import androidx.core.view.isVisible
import com.loitp.annotation.IsFullScreen
import com.loitp.annotation.LogTag
import com.loitp.core.base.BaseActivityFont
import com.loitp.core.ext.setSafeOnClickListenerElastic
import com.loitp.views.answerView.LAnswerView
import kotlinx.android.synthetic.main.a_answer_view.*
import vn.loitp.R

@LogTag("AnswerViewActivity")
@IsFullScreen(false)
class AnswerViewActivity : BaseActivityFont() {

    override fun setLayoutResourceId(): Int {
        return R.layout.a_answer_view
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setupViews()
        // use xml
        useXML()
        // use JAVA codes
        useJava()
    }

    private fun setupViews() {
        lActionBar.apply {
            this.ivIconLeft.setSafeOnClickListenerElastic(
                runnable = {
                    onBaseBackPressed()
                }
            )
            this.ivIconRight?.isVisible = false
            this.tvTitle?.text = AnswerViewActivity::class.java.simpleName
        }
    }

    private fun useXML() {
        lAnswerView1.setNumber(1)
        lAnswerView1.setOnAnswerChange(object : LAnswerView.OnAnswerChange {
            override fun onAnswerChange(view: LAnswerView?, index: Int) {
                showShortInformation("Click: $index")
            }
        })
        lAnswerView1.activeChar = 'A'
        // answerView1.resize(2)

        lAnswerView2.setNumber(2)
        lAnswerView2.setOnAnswerChange(object : LAnswerView.OnAnswerChange {
            override fun onAnswerChange(view: LAnswerView?, index: Int) {
                showShortInformation("Click: $index")
            }
        })
    }

    private fun useJava() {
        for (i in 0..9) {
            val lAnswerView = LAnswerView(this)
            lAnswerView.init(
                number = i + 3,
                numberOfAnswer = 6,
                isShowNumber = true,
                isCanCancelAnswer = true,
                isChangeOnClick = true,
                isShowTextWhenActive = true
            )
            lAnswerView.setOnAnswerChange(object : LAnswerView.OnAnswerChange {
                override fun onAnswerChange(view: LAnswerView?, index: Int) {
                    showShortInformation("Click: $index")
                }
            })
            ll.addView(lAnswerView)
        }
    }
}